packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -330,11 +375,11 @@ export class EVMAAWallet<B extends EVMBlockchainIncludingTestnet = EVMBlockchain
                 throw new Error("New version info not found");
             }
 
-            const enableSig = await this.kernelClient.account.kernelPluginManager.getPluginEnableSignature(
-                this.kernelClient.account.address
+            const enableSig = await this.kernelClient.account?.kernelPluginManager.getPluginEnableSignature(
+                this.kernelClient.account?.address
```

Comment:
above on line 359 we have the `== null` check, we should be able to remove these `?` now, right?
```suggestion
            const enableSig = await this.kernelClient.account.kernelPluginManager.getPluginEnableSignature(
                this.kernelClient.account.address
```

we also use `this.kernelClient.account` a bunch of places in this function, we may want to do
```
const { account } = this.kernelClient
if (account == null) {
  throw Error